core/chains/evm/chain_set.go

```diff
@@ -24,7 +24,8 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
-var ErrNoChains = errors.New("no chains loaded, are you running with EVM_DISABLED=true?")
+// ErrNoChains indicates that no chains have been started
+var ErrNoChains = errors.New("no chains loaded")
```

Comment:
Will this always be wrapped/logged with something that specifies `evm`? Or would that be important to include here?